virtual-programs/receipt_printer.folk

```diff
@@ -0,0 +1,56 @@
+namespace eval EscPos {
+    proc printProgram {id code address} {
+        set folkFile [open "$::env(HOME)/folk-printed-programs/$id.folk" w]
+        puts $folkFile $code
+        close $folkFile
```

Comment:
When printing programs the traditional way, we abort the print if the program already exists. That's not how I remember it but that's the way the code looks. I could add that here but it's nice to be able to reprint a program if the code gets updated.